js/lib/analytics.js

```diff
@@ -110,7 +110,11 @@ if(lqx && !('analytics' in lqx)) {
 				if(opts.enabled) {
 					lqx.log('Initializing `analytics`');
 
-					// Load Google Analytics
+					// Load only Google Analytics 4
+					if(!opts.usingGTM && opts.createParams && opts.createParams.default && !opts.createParams.default.trackingId && opts.createParams.default.measurementId) {
```

Comment:
this shouldn't include the condition !opts.createParams.default.trackingId
we want to load ga4 whenever we have measurement id value